src/Monitor/Monitor/Alerts/AddAzureRmMetricAlertRuleV2Command.cs

```diff
@@ -98,6 +98,12 @@ public class AddAzureRmMetricAlertRuleV2Command : ManagementCmdletBase
         [Alias("Criteria")]
         public List<IPSMultiMetricCriteria> Condition { get; set; }
 
+        /// <summary>
+        /// Gets or sets the AutoMitigate  parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipeline = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The flag that indicates whether the alert should be auto resolved or not")]
+        public Boolean AutoMitigate { get; set; } = true;
```

Comment:
Hi @BethanyZhou, thanks for the reference.
I consulted our PM, since that's the param name used in alerts other client API's (like CLI) we prefer to keep it as is.
If it's an issue (per the best practices) - we will change it.

Reply:
Hi @hutzleo , thank you for your response. Bool is acceptable for us.